simulation/samples/JavaSample/src/main/java/frc/robot/Robot.java

```diff
@@ -15,117 +12,164 @@
-    System.out.println("Auto selected: " + m_autoSelected);
-  }
-
-  /** This function is called periodically during autonomous. */
-  @Override
-  public void autonomousPeriodic() {
-
-    m_Spark.set(0.5);
-    m_SparkMax.set(1.0);
-    m_Talon.set(-1.0);
-
-    switch (m_autoSelected) {
-      case kCustomAuto:
-        // Put custom auto code here
-        break;
-      case kDefaultAuto:
-      default:
-        // Put default auto code here
-        break;
+    private static final String kDefaultAuto = "Default";
+    private static final String kCustomAuto = "My Auto";
+    private String m_autoSelected;
+    private final SendableChooser<String> m_chooser = new SendableChooser<>();
+
+    private CANSparkMax m_SparkMax1 = new CANSparkMax(1, MotorType.kBrushless);
+    private CANSparkMax m_SparkMax2 = new CANSparkMax(2, MotorType.kBrushless);
+    private CANSparkMax m_SparkMax3 = new CANSparkMax(3, MotorType.kBrushless);
+    private CANSparkMax m_SparkMax4 = new CANSparkMax(4, MotorType.kBrushless);
+    private CANSparkMax m_SparkMax5 = new CANSparkMax(5, MotorType.kBrushless);
+    private CANSparkMax m_SparkMax6 = new CANSparkMax(6, MotorType.kBrushless);
+    //private com.autodesk.synthesis.ctre.TalonFX m_Talon = new com.autodesk.synthesis.ctre.TalonFX(2);
+
+    /**
+     * This function is run when the robot is first started up and should be used
+     * for any
+     * initialization code.
+     */
+    @Override
+    public void robotInit() {
+        m_chooser.setDefaultOption("Default Auto", kDefaultAuto);
+        m_chooser.addOption("My Auto", kCustomAuto);
+        SmartDashboard.putData("Auto choices", m_chooser);
+    }
+
+    /**
+     * This function is called every 20 ms, no matter the mode. Use this for items
+     * like diagnostics
+     * that you want ran during disabled, autonomous, teleoperated and test.
+     *
+     * <p>
+     * This runs after the mode specific periodic functions, but before LiveWindow
+     * and
+     * SmartDashboard integrated updating.
+     */
+    @Override
+    public void robotPeriodic() {
+    }
+
+    /**
+     * This autonomous (along with the chooser code above) shows how to select
+     * between different
+     * autonomous modes using the dashboard. The sendable chooser code works with
+     * the Java
+     * SmartDashboard. If you prefer the LabVIEW Dashboard, remove all of the
+     * chooser code and
+     * uncomment the getString line to get the auto name from the text box below the
+     * Gyro
+     *
+     * <p>
+     * You can add additional auto modes by adding additional comparisons to the
+     * switch structure
+     * below with additional strings. If using the SendableChooser make sure to add
+     * them to the
+     * chooser code above as well.
+     */
+    @Override
+    public void autonomousInit() {
+        m_autoSelected = m_chooser.getSelected();
+        //m_autoSelected = SmartDashboard.getString("Auto Selector", kDefaultAuto);
```

Comment:
Leaving this for the future?

Reply:
that was some testing @PepperLola was doing